clusters.py

```diff
@@ -7,7 +9,12 @@
 of the user.
 """
 spreadsheet_v1_path = 'https://docs.google.com/spreadsheets/d/119_-qrfzGXwV1YBUJdBzvtAQTZnl-xwN7hD9FK5SWfU/edit#gid=84023254'
+features_before = [ "AgeAtTestDate", "R250", "R500", "R1000", "R2000", "R3000", "R4000", "R6000", "R8000",
```

Comment:
Good, but I'm not sure what this name means.  Before what?

the AC and BC fields make sense.  Are we using the other scores?

Not sure what the right answer is... perhaps features_v1?

Reply:
I used the terms features_before and features_after to distinguish the features that will have new columns after HLClassification() and Kmeans cluastering.

Now that I see, features_initial will make more sense. I will leave a comment also nearby.